lib/opensusebasetest.pm

```diff
@@ -257,6 +257,33 @@ sub handle_uefi_boot_disk_workaround {
     wait_screen_change { send_key 'ret' };
 }
 
+=head2 rewrite_static_svirt_network_configuration
+
+Rewrite the static network configuration within a SUT over the svirt console
+based on the worker specific configuration to allow reuse of images created on
+one host and booted on another host. Can also be used for debugging in case of
+consoles relying on remote connections to within the SUT being blocked by
+malfunctioning network or services within the SUT.
+
+Relies on the C<$pty> variable set in the remote svirt shell by C<utils::save_svirt_pty>.
+
+Also see poo#18016 for details.
+=cut
+sub rewrite_static_svirt_network_configuration {
```

Comment:
I am not so sure about this one because the machine is not persistant but the svirt shell is. I would keep it as is for now as it looks more robust to me to force overwriting the same IP instead of not doing it at all but this could cause some confusion to the reader. We will touch this code for sure anyway later.